src/main.rs

```diff
@@ -348,9 +348,9 @@ fn main() -> anyhow::Result<()> {
             }
         });
 
-        runtime_handle.block_on(async {
-            toc_arc.resume_resharding_tasks().await;
-        });
+        // runtime_handle.block_on(async {
+        //     toc_arc.resume_resharding_tasks().await;
+        // });
```

Comment:
Disable resuming resharding driver task when starting Qdrant. I've missed it when disabling resharding driver earlier. 😅

Reply:
Done.